app/Repositories/UserStore.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace App\Repositories;
+
+use App\Repositories\Contracts\UserStoreInterface;
+use App\Facades\CurlService;
+use App\Models\User;
+use DB;
+
+class UserStore implements UserStoreInterface
+{
+
+    public function get($user)
+    {
+        if (strtolower(env('APP_ENV')) <> 'local') {
+            $response = CurlService::sendUsersRequest($user->global_id);
+            $users = array_map(
+                function ($userFunc) {
+                    $r['first_name'] = $userFunc->name;
+                    $r['last_name'] = $userFunc->surname;
+                    $r['email'] = $userFunc->email;
+                    $r['city'] = $userFunc->city;
+                    $r['country'] = $userFunc->country;
+                    $r['birthday'] = $userFunc->birthday;
+                    $r['local_id'] = $userFunc->id;
+                    $r['avatar'] = $userFunc->avatar;
+                    return $r;
+                },
```

Comment:
I guess it could be possible to move mapper to separate function to prevent repetitive code